movement/watch_faces/complication/counter_face.c

```diff
@@ -48,12 +54,25 @@ bool counter_face_loop(movement_event_t event, movement_settings_t *settings, vo
 
     switch (event.event_type) {
         case EVENT_ALARM_BUTTON_UP:
+            watch_buzzer_abort_sequence; //abort running buzzer sequence when counting fast
```

Comment:
This call and the other call on line 68 need to be `watch_buzzer_abort_sequence();`

Without the parentheses you aren’t actually calling a function.